src/ci/BitbucketPipelinesAdapter.js

```diff
@@ -33,20 +33,29 @@ const BitbucketPipelinesAdapter: CIAdapter = (config: Config) => {
     processStatusWebhook(body: JSONValue): StatusEvent | null {
       if (
         !body ||
-        !body.commit_status ||
-        !body.commit_status.state ||
-        !body.commit_status.url ||
-        typeof body.commit_status.url !== 'string'
+        !body.data ||
+        !body.data.commit_status ||
```

Comment:
This is super strange.

Part of the mystery around the strange build_status' was solved, but it actually doesnt explain how this was ever working.